### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by AdamKyle** [beyondcode/laravel-websockets#1038] [General]
The last update was 9-10 months ago and there are a lot of issues open and PR's open. I am curious because I currently depend on it for my application but have not seen any updates. Last I heard 2.0 was in the works, but again - no updates.

Thoughts?

**Comment by mpociot** [comment]
Hi!
I started fixing a lot of memory leaks, performance issues as well as added some new features like being able to use MySQL as a an app provider. 
I am also going through the issues, so actually there's a lot going on right now. 
Admittedly this is all happening on the v2 side of things. But I plan on launching v2 very very soon!

**Reply by AdamKyle** [reply]
Ok, thank you for the update.

**Reply by chrispage1** [reply]
Looking forward to V2 - the new features look great - thank you Marcel!

**Reply by Wernke96** [reply]
I'm looking forward to V2 as well.

**Reply by imhannan** [reply]
@mpociot How long it will take releasing new version of this package?

**Reply by stratboy** [reply]
Yes, is it still in development?

**Reply by parallels999** [reply]
very very soon! 😄

**Reply by Waleedx249i** [reply]
😂 friends this my first time to use this program and i have issues can anyone help??

**Comment by parallels999** [comment]
Almost

**Comment by imhannan** [comment]
What is the latest update? Will we get new version?

**Reply by parallels999** [reply]
No releases, no support

**Comment by Neodark7** [comment]
"very very soon" 🤡

**Comment by stratboy** [comment]
Is this program safe for production? Will it be still developed?

**Reply by parallels999** [reply]
> Will it be still developed?

There has been no movement for a long time

**Reply by chrispage1** [reply]
Unfortunately there hasn't been any updates in a while. We've since moved to self hosting a Soketi instance which is a drop-in Pusher alternative.

**Reply by cerw** [reply]
@mpociot can you tell us if you still maintaining this or we should move to new solutions ? thank you
